consensus/src/quorum_store/quorum_store_wrapper.rs

```diff
@@ -53,28 +55,40 @@ pub struct QuorumStoreWrapper {
     // For ensuring that batch size does not exceed QuorumStore limit.
     quorum_store_max_batch_bytes: u64,
     last_end_batch_time: Instant,
+    db: Arc<QuorumStoreDB>,
 }
 
 impl QuorumStoreWrapper {
     pub fn new(
         epoch: u64,
+        db: Arc<QuorumStoreDB>,
         mempool_tx: Sender<QuorumStoreRequest>,
         quorum_store_sender: TokioSender<QuorumStoreCommand>,
         mempool_txn_pull_timeout_ms: u64,
         mempool_txn_pull_max_count: u64,
         quorum_store_max_batch_bytes: u64,
     ) -> Self {
+        if epoch > 0 {
+            db.delete_batch_id(epoch - 1).expect("Could not delete from db");
```

Comment:
should delete everything that's lower than epoch, 
maybe can rename it to clean_epochs... something batch_ids, idk :-)